### DISCUSSION TITLE
Restore original Start Icon Windows 11

### DISCUSSION THREAD

**Discussion by dartoi** [valinet/ExplorerPatcher#142] [Q&A]
Dear All,

how could I restore the original Start Icon Windows 11 with dxgi.dll applied?

Regards.

**Comment by sisericm** [comment]
Its a bit of a pain but you could use openshell. 
Let openshell handle the start menu and set a custom icon using the win 11 icon.
For example [https://www.deviantart.com/aquasholdmt8/art/Windows-11-Start-Button-Icon-886655201](url)

**Reply by techlogik** [reply]
Your link doesn't work.

**Reply by Gaurav-Original-ClassicShellTester** [reply]
Open Shell is always an option and you get a tremendous choice of existing Start buttons including animated ones: http://www.classicshell.net/forum/viewforum.php?f=18 however that DeviantArt image won't work since it's not in the proper 3 image format required for a Start button image. Here I threw one together quickly: http://www.classicshell.net/forum/viewtopic.php?f=18&t=8582 . Anyone is free to improve upon it:

**Reply by sisericm** [reply]
@techlogik Apologies for the dud link. I answered in too much of a hurry and should have checked it more thoroughly. Thanks @Gaurav-Original-ClassicShellTester for dropping a decent working one in.
I was trying to find these earlier but was too pushed for time.

![startdarkv2](https://user-images.githubusercontent.com/8255994/138494299-c17d7517-dd87-4ade-8283-e5c6ed0bbf84.png)
![startdark](https://user-images.githubusercontent.com/8255994/138494311-1659a536-f4b0-44f6-88a9-d8c8c96cb1b6.png)

![lightfixed](https://user-images.githubusercontent.com/8255994/138494321-c9f2fc1b-339a-40f9-b9aa-562a0547b541.png)
![light](https://user-images.githubusercontent.com/8255994/138494330-5e794acf-9c41-4e1f-bce6-d12c8a189ff3.png)

**Reply by Gaurav-Original-ClassicShellTester** [reply]
Note that GitHub seems to destroy the alpha transparency of PNG files so maybe attaching them to the Classic Shell forum or imgur or some place else is a better idea.

**Reply by dartoi** [reply]
@valinet could you add in "Properties" an option to select Window 10 or Windows 11 logo start menu? (and a script to select autoupdate the file "dxgi.dll")

Thanks a lot.

**Reply by valinet** [reply]
When the app will have that capability, I will put.

Script to replace automatically: https://github.com/valinet/ExplorerPatcher/discussions/183

Updates over the Internet maybe in a future release. I only have 24 hours a day...

Thanks

**Reply by sisericm** [reply]
> 
> 
> Note that GitHub seems to destroy the alpha transparency of PNG files so maybe attaching them to the Classic Shell forum or imgur or some place else is a better idea.

Thanks for the head's up. I didn't know.

**Reply by sisericm** [reply]
Here is a zip containing the icons.

[start buttons.zip](https://github.com/valinet/ExplorerPatcher/files/7403427/start.buttons.zip)

**Reply by Gaurav-Original-ClassicShellTester** [reply]
> Here is a zip containing the icons.
> 
Nice. Can you make high DPI versions? These days, at least some of my PCs are running 1080p with 150% DPI scaling so high res buttons would be awesome to have. Although I am sure over time, they will crop up at sites like DeviantArt etc :)

**Reply by klydeinside** [reply]
> When the app will have that capability, I will put.
> 
> Script to replace automatically: #183
> 
> Updates over the Internet maybe in a future release. I only have 24 hours a day...

You're using the ellipsis wrong and it makes you sound like a condescending dick to someone simply asking about features. Though, from the way you type, you probably are.

**Reply by valinet** [reply]
@itzklyde https://youtu.be/nfWlot6h_JM

Also, thanks for showing how a proper reply should look. I will take it into consideration.

**Reply by BraINstinct0** [reply]
@itzklyde 
> You're using the ellipsis wrong and it makes you sound like a condescending dick to someone simply asking about features. Though, from the way you type, you probably are.

Well, welcome to Opensource - when you find an issue, you either report, suggest a fix, make a PR, or if all doesn't work:fork the project and maintain patches by oneself. (Not raise a pitchfork against a developer😉)
